src/core/services/aria/aria.js

```diff
@@ -1,11 +1,60 @@
+/**
+ * @ngdoc module
+ * @name material.core.aria
+ * @description
+ * Aria Expectations for ngMaterial components.
+ */
+angular
+  .module('material.core')
+  .provider('$mdAria', MdAriaProvider);
+
+/**
+ * Whether we should show ARIA warnings in the console, if labels are missing on the element
+ * By default the warnings are enabled
+ */
+var showWarnings = true;
+
+
+/**
+ * @ngdoc service
+ * @name $mdAriaProvider
+ * @module material.core.aria
+ *
+ * @description
+ *
+ * Modify options of the `$mdAria` service, which will be used by most of the Angular Material components.
+ **
+ *
+ * You are able to disable `$mdAria` warnings, by using the following markup.
+ * <hljs lang="js">
+ *   app.config(function($mdAriaProvider) {
+ *     // Globally disables all ARIA warnings.
+ *     $mdAriaProvider.disableWarnings();
+ *   });
+ * </hljs>
+ *
+ */
+function MdAriaProvider() {
 
-angular.module('material.core')
-  .service('$mdAria', AriaService);
+  return {
+    disableWarnings: disableWarnings,
+    $get: MdAriaService
+  };
+
+  /**
+   * @ngdoc method
+   * @name $mdAriaProvider#disableWarnings
+   * @param {boolean} disable Whether to show `$mdAria` console warnings. Default value is `false`.
+   */
+  function disableWarnings(disable) {
+    showWarnings = !disable || false;
```

Comment:
@ThomasBurleson I actually added that to be able to reset the provider in the tests (we should not make changes for the tests to pass)

I will try to find a better solution.